pkg/features/kube_features.go

```diff
@@ -425,6 +425,7 @@ const (
 
 	// owner: @wk8
 	// alpha: v1.14
+	// beta: v1.15
```

Comment:
This change is no longer true. This should stay as is.

Reply:
👍